test/e2e/bootstrappable/snapshots.py

```diff
@@ -0,0 +1,60 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+#	 http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+from dataclasses import dataclass
+
+import boto3
+import time
+from acktest.bootstrapping import Bootstrappable
+from acktest.resources import random_suffix_name
+
+
+@dataclass
+class Snapshots(Bootstrappable):
```

Comment:
Great job writing a `Bootstrappable` type! Really happy to see it being used.

This `Bootstrappable` is actually bootstrapping a `MemoryDB` cluster - so should probably be called `Cluster`

```suggestion
class Cluster(Bootstrappable):
```